.travis.yml

```diff
@@ -14,9 +14,16 @@ after_success:
   - npm run doc
 
 deploy:
-  provider: pages
+- provider: pages
   skip_cleanup: true
   local_dir: site
   github_token: $GITHUB_TOKEN
   on:
     branch: master
+- provider: npm
+  email: linuxwolf@outer-planes.net
```

Comment:
Should this email some Lockbox group address? (Disclaimer: I have no idea what the email address does or if it sends emails, I'm just generally frightened of specificity.)

Reply:
I wish it could be, but npm is not set up for that.  I can (and am) changing to a \@mozilla.com email address, but it still has to be an individual.